canonical-web-and-design/jaas.ai
# Update the store hero section

## Done
Updated the store hero section to matches the latest designs.
_Just need the light versions of the expert logos from @spencerbygraves_

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/store
- Check the hero/top section of the site matches [the new design](https://app.zeplin.io/project/5cf8ca699ee7bf1df9082012/screen/5cf8ca4354e6df1dffcd7cfc)

## Details
Fixes https://github.com/canonical-web-and-design/jaas.ai/issues/326

## Screenshots
![Screenshot_2019-07-01 Jujucharms Juju(1)](https://user-images.githubusercontent.com/1413534/60468767-986c5a80-9c52-11e9-825b-c0e550313484.png)
